docs/zilevm/protocol-opcodes.md

```diff
@@ -0,0 +1,18 @@
+---
+id: protocol-opcodes
+title: Opcodes
+keywords:
+  - opcodes
+description: ZILEVM Opcodes
+---
+
+---
+
+## Opcodes
```

Comment:
I think this page could do with an introductory paragraph describing what it is talking about. It isn't really clear to me. I can probably guess, but I don't think I should have to.

Reply:
Any suggestions for copy?